content/tutorial/tutorial.md

```diff
@@ -1126,7 +1133,7 @@ class Game extends React.Component {
   }
 ```
 
-Next, we'll define the `jumpTo` method in Game to update that `stepNumber`. We also set `xIsNext` to true if the number that we're changing `stepNumber` to is even:
+अगला, हम गेम में `jumpTo` पद्धति को परिभाषित करेंगे ताकि यह अपडेट किया जा सके कि `stepNumber`। यदि हम जिस नंबर को बदल रहे हैं, तो हम `xIsNext` को भी सही सेट करते हैं, यहाँ तक कि `stepNumber` को भी:
```

Comment:
पद्धति -> method in hindi?

Reply:
अगला, हम गेम में `jumpTo` पद्धति को परिभाषित करेंगे ताकि यह अपडेट किया जा सके कि `stepNumber`।  does not make sense gramatically

Reply:
> अगला, हम गेम में `jumpTo` पद्धति को परिभाषित करेंगे ताकि यह अपडेट किया जा सके कि `stepNumber`। does not make sense gramatically

```
अब, हम `stepNumber` को अपडेट करने के लिए Game में `jumpTo` मेथड लिखेंगे
```

Reply:
cool